web-animations-1/Overview.bs

```diff
@@ -3926,6 +3967,7 @@ The <code>AnimationTimeline</code> interface {#the-animationtimeline-interface}
 [Exposed=Window]
 interface AnimationTimeline {
     readonly attribute double? currentTime;
+    readonly attribute TimelinePhase phase;
```

Comment:
There should probably be a description of the member somewhere that refers back to the definition in the model.